src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Fields/ContentFieldsProvider.cs

```diff
@@ -107,9 +114,10 @@ public FieldType GetField(ISchema schema, ContentPartFieldDefinition field, stri
 
         return new FieldType
         {
-            Name = customFieldName ?? field.Name,
+            Name = customFieldName ?? schema.NameConverter.NameForField(field.Name, null),
             Description = fieldDescriptor.Description,
             Type = fieldDescriptor.FieldType,
+            ResolvedType = fieldDescriptor.ResolvedType,
```

Comment:
If I understand the changes here correctly, we can get rid of the assignment `Type = FieldType`. The type has been replaced by `ResolvedType` because the fields are added during schema initialization. Is that assumption correct?

Reply:
done